packages/alt-ergo-free/alt-ergo-free.2.2.0/opam

```diff
@@ -0,0 +1,80 @@
+opam-version: "2.0"
+name: "alt-ergo-free"
+version: "2.2.0"
+
+maintainer: "alt-ergo@ocamlpro.com"
+authors: "Alt-Ergo developers"
+license: "Apache-2.0"
+homepage: "http://alt-ergo.ocamlpro.com/"
+dev-repo: "git+https://github.com/OCamlPro/alt-ergo.git"
+bug-reports: "https://github.com/OCamlPro/alt-ergo/issues"
+
+build:
+[
+  ["autoconf"]
+  ["./configure" "-prefix" "%{prefix}%"]
+	[make]
+]
+
+install:
+[
+	[make "install" "MANDIR=%{man}%"]
+]
+
+remove: [
+        ["autoconf"]
+        ["./configure" "-prefix" "%{prefix}%"]
+	[make "uninstall" "MANDIR=%{man}%"]
+]
```

Comment:
The `remove` field isn't necessary anymore since opam 2.0 unless you need to remove files that have been created after the installation (config files for instance).

Reply:
Thanks. The `remove` field will be removed for the next release, should i do something for this release ?

Reply:
No it should be fine. Thanks a lot!